### PR TITLE
update devDependencies and remove unused _ variable

### PR DESCRIPTION
**Pull request by dvalentiate**

### CODE REVIEW

**Comment by dvalentiate** [comment]
This repo has been long neglected and needs a little love.

File: .travis.yml [hunk 1]
```diff
@@ -4,11 +4,9 @@ branches:
         - master
         - qc
 node_js:
-    - "0.12.5"
-    - "4"
-    - "6"
```
Comment (MitMaro) [inline .travis.yml]: If this is a statement of dropping support for these versions than the package should be a major bump.

**Review by MitMaro — APPROVED** [review]
Left one comment, but I am okay with the decision on it either way.

File: .travis.yml (hunk 1 above)
Comment (dvalentiate) [reply .travis.yml]: So I created a PR `https://github.com/CoursePark/NestHydrationJS/pull/34` with only an empty commit on master and it fails in 0.12. So I'm not sure who it passed previously or if it did but any reasonable path forward involves dropping 0.12. It is just a question of whether this must be a major bump. As it is maintaining 0.12 support make security updates impossible.

Comment (dvalentiate) [reply .travis.yml]: went with version 2

**Pull request merged**